main.py

```diff
@@ -0,0 +1,6 @@
+from itemloaders import ItemLoader
+from itemloaders.processors import MapCompose, TakeFirst
+
+
+class CustomItemLoader(ItemLoader):
+    name_out = TakeFirst()
```

Comment:
I don’t see this file being used anywhere. A debug leftover, maybe?

Reply:
Yes :sweat_smile: